.github/workflows/ci.yml

```diff
@@ -37,6 +39,7 @@ jobs:
           - php-version: 8.0
           - php-version: 8.1
```

Comment:
you can drop 8.1 and 8.0 from these matrixes 🙏

Reply:
Also update composer.json to say `"php": "^8.2"`

Reply:
Done!